examples/stdrun.hoc

```diff
@@ -0,0 +1,755 @@
+objref cvode
```

Comment:
Add reference to what this example hoc file is based on. From the Neuron reference doc?
Useful for instance when that one is updated. to be able to compare.

Reply:
Same for the other .hoc file here.